Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Visualization/FormDumpTests.cs

```diff
@@ -36,20 +37,27 @@ public new void SetUp()
         public void Should_BePossibleTo_SaveFormDump_WithDefaultName()
         {
             var form = new WebForm();
-            var pathToDump =
-                new DirectoryInfo(Path.Combine(PathToDumps, form.Name.Replace("/", " ")));
-            if (pathToDump.Exists)
-            {
-                pathToDump.Delete(true);
-                pathToDump.Refresh();
-            }
-            Assert.AreEqual(0, pathToDump.Exists ? pathToDump.GetFiles().Length : 0, "Dump directory should not contain any files before saving");
+            var pathToDump = CleanUAndGetpPathToDump(form.Name.Replace("/", " "));
+
             Assert.DoesNotThrow(() => form.Dump.Save());
             pathToDump.Refresh();
             DirectoryAssert.Exists(pathToDump);
             Assert.Greater(pathToDump.GetFiles().Length, 0, "Dump should contain some files");
         }
 
```

Comment:
Please add some test(s) to ensure that custom format could be used for images, and images are actualy saved in this format (for example, that dumpFile.jpg is not in png format, but a valid file in jpg format.